packages/web-app-files/src/mixins/actions/rename.js

```diff
@@ -81,13 +81,16 @@ export default {
           : this.$gettext('Rename file %{name}'),
         { name: resources[0].name }
       )
+      const resourceName = resources[0].name
+      const nameWithoutExtensionLength = resourceName ? resourceName.split('.')[0].length : -1
```

Comment:
This doesn't work for file names that have more than one `.`. It is possible that the filename part as any amount of `.`s. And there are also some file extensions  that consist of more than one segment, e.g. `.tar.gz`. For a start I'd be fine that you select everything until the last dot. That would break `archive.tar.gz` as it will select too much (`archive.tar` instead of `archive`). In a followup PR you could add a mapping table for file extensions that are known to have more than one segment (like e.g. the beforementioned `.tar.gz`).

Reply:
I see, yes thought of that too but realized we dont have the file extension on the resource, but creating a mapping file sounds good to me. Will do that in a second PR.